### DISCUSSION TITLE
Dexcom G7 to wahoo bolt

### DISCUSSION THREAD

**Discussion by barbmesser** [NightscoutFoundation/xDrip#3326] [Q&A]
I've been successful getting G6 data on my wahoo bolt.  However, I can't get the wahoo to accept the Glucose field now that I use the G7.  After "start usb configuration / allow xdrip access to ELEMNT (okay) / permission granted / all stages successful / wait for device to reboot" the wahoo does reboot but without the Glucose field.  Any suggestions?

**Comment by hpaegel** [comment]
I am interested in this as well since I am going to receive my G7 system soon. Maybe @jamorham knows something about this? It would be helpful to get an idea on why this doesn't work and how this might be changed.

**Comment by Navid200** [comment]
I don't think  the interaction with Wahoo cares about what sensor the readings come from.

**Comment by Gren69** [comment]
Hi. Is this possible on iOS ?

**Reply by Navid200** [reply]
xDrip only runs on Android.
